config/routes.rb

```diff
@@ -1,20 +1,24 @@
 Rails.application.routes.draw do
 
-  root to: 'home#index'
-  resources :cards
-  resources :reviews, only: [:new, :create]
-  resources :decks do
-    get :set_current, on: :member
+  root to: 'guest/home#index'
+  scope module: "dashboard" do
+    resources :cards
+    resources :reviews, only: [:new, :create]
+    resources :decks do
+      get :set_current, on: :member
+    end
+    resource  :profile, controller: :profile, only: [:edit,:update]
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Space missing after comma.